README.md

```diff
@@ -318,16 +318,16 @@ Instead of using config in the HOC, you can put it in the Component class. This
 - initialValues: `{ propName: value }`
 - validationSchema: `{ propName: (value, props) => 'error'||{} }`
 - validateOnBlur: `boolean`. Default: `true`
-- validateOnChang: `boolean`. Default: `true`
+- validateOnChange: `boolean`. Default: `true`
 
 ##### Props
 - values: `object`
 - errors: `object`
 - touched: `object`
 - isValid: `boolean`
 - handleSubmit: `function`
-- handleChange: `(path || event, value) => {}`. event: `Event: {target: {name || id, value}} || CustomEvent: {detail: {name || id, value}}`
-- handleBlur: `(path || event) => {}`. event: `Event: {target: {name || id}} || CustomEvent: {detail: {name || id}}`
+- handleChange: `(event, value) => {}`. event: `Event: {target: {name || id, value}} || CustomEvent: {detail: {name || id, value}}`
```

Comment:
I noticed that new function does not accept "value" argument

Reply:
See my comments in `withForm`.
TL;DR I am not a big fan of function behavior changing according to the type of the params and would prefer 2 different methods.

Reply:
I'm talking about what is worth removing `value` from arguments here - it is not actual anyway.